### PR TITLE
Update parameter explanations

### LINKED ISSUE #7511: Yolov8 augment = False still augmenting train data
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

_No response_

### Bug

hey everyone, 

I load the yolo model, disable as many augment values as possible. The output folder is still showing augemnts. 

Here is my code 

yolo_code

Any the output train batch image;

train_batch0

### Environment

Ultralytics YOLOv8.1.0 🚀 Python-3.10.12 torch-2.1.0+cu121 CUDA:0 (Tesla T4, 15102MiB)

### Minimal Reproducible Example

import os

from ultralytics import YOLO

DATA_DIR = source_dir_morl_s2fs

model = YOLO("yolov8n-cls.pt")  # load a pretained model

results= model.train(data=DATA_DIR,
                     epochs=20, batch = 128, imgsz=310,
                     augment =False,
                     hsv_h =0, hsv_s =0, hsv_v =0,degrees =0, translate =0, scale =0,
                     shear =0, flipud =0,fliplr=0,mixup =0,agnostic_nms =False,cos_lr =False,
                     mosaic = 0.0,
                     save = True, 
                     save_dir = source_dir + 'power_image/saved_models/runs4/classify/train/',
                     save_period = 1)

### Additional

_No response_

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

### LINKED ISSUE #8987: Augementations are applied anyway in "classify, train mode" even when it's disabled
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Train

### Bug

Hello, It seems like the model is using augmentations even when I set `augment: false` in  'ultralytics/cfg/default.yaml'.
train_batch0_1_c0da9b9c9fbf05196de1

### Environment

```
Ultralytics YOLOv8.1.28 🚀 Python-3.10.13 torch-2.2.1+cu121 CUDA:0 (NVIDIA GeForce RTX 4090, 24209MiB)
Setup complete ✅ (32 CPUs, 62.5 GB RAM, 120.3/960.2 GB disk)

OS                  Linux-6.5.0-25-generic-x86_64-with-glibc2.35
Environment         Linux
Python              3.10.13
Install             pip
RAM                 62.51 GB
CPU                 Intel Core(TM) i9-14900KF
CUDA                12.1

matplotlib          ✅ 3.8.3>=3.3.0
opencv-python       ✅ 4.9.0.80>=4.6.0
pillow              ✅ 10.2.0>=7.1.2
pyyaml              ✅ 6.0.1>=5.3.1
requests            ✅ 2.31.0>=2.23.0
scipy               ✅ 1.12.0>=1.4.1
torch               ✅ 2.2.1>=1.8.0
torchvision         ✅ 0.17.1>=0.9.0
tqdm                ✅ 4.66.2>=4.64.0
psutil              ✅ 5.9.8
py-cpuinfo          ✅ 9.0.0
thop                ✅ 0.1.1-2209072238>=0.1.1
pandas              ✅ 2.2.1>=1.1.4
seaborn             ✅ 0.13.2>=0.11.0
```

### Minimal Reproducible Example

```bash
yolo task=classify mode=train model=yolov8n-cls.pt data=dataset_by_class epochs=500 imgsz=640 patience=100 plots=True cache=memory
```

### Additional

_No response_

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

### PR DESCRIPTION
**Pull request by fcakyon**
Improves explanations of classification parameters in default config file.

closes https://github.com/ultralytics/ultralytics/pull/9026

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Improve image augmentation for better AI model training. 📈

### 📊 Key Changes
- **Added `crop_fraction`** to the list of augmentations, allowing images to be cropped to a fraction of their size. This helps focus on central features and reduces background distractions.
- **Adjusted `erasing` range**: Now caps at 0.9, refining how objects might be randomly erased during training to better focus model learning.
- **Documentation and default config updates** to reflect these changes, ensuring users can easily adjust these settings for their training needs.

### 🎯 Purpose & Impact
- **Enhance Model Generalization**: By introducing more diverse training scenarios like cropping and adjusted erasing, models can learn to focus on relevant features more efficiently.
- **Reduce Overfitting**: These changes can help models perform better on unseen data by reducing the likelihood of memorizing irrelevant background details.
- **Flexible and Tailored Training**: Adding `crop_fraction` and adjusting `erasing` gives developers more tools to tailor training to specific requirements, potentially leading to improved model accuracy and robustness.

These updates make the AI training process more adaptable and focused, likely leading to better-performing models across a wider range of conditions. 🚀

### CODE REVIEW

**Comment by glenn-jocher** [comment]
@fcakyon thanks for the PR!

If argument descriptions are updated, they generally need updating also in 3 places:

- Docs page they are in, i.e. for these args it would be train page at https://docs.ultralytics.com/modes/train/
- Usage/config page at https://docs.ultralytics.com/usage/cfg/
- default.yaml (already done here)

**Comment by glenn-jocher** [comment]
@fcakyon wait I was looking at #9026. Should I close that PR and merge this one instead?

**Comment by fcakyon** [comment]
> @fcakyon wait I was looking at #9026. Should I close that PR and merge this one instead?

Yes, please @glenn-jocher.
